skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -512,6 +513,12 @@ def test_inference(
     assert np.allclose(output, y_pred)
 
 
+def test_get_model_output_deprecated():
```

Comment:
If, in the test above, you add the `match` to `pytest.warns`, this test would be completely redundant, right? This might be a worthwhile change, so that we avoid hitting the endpoint even more often. If you want to leave this as a separate test, it should have the `@pytest.mark.network` and `@pytest.mark.inference` decorators markers, right?

Reply:
Here I'm not passing any token and not passing a valid model name, so the function would fail anyway (hence the `pytest.raises(Exception)`, therefore I don't think it even needs a network marker, and it can be in all situations and not only with a specific commit message or a merker for pytest.

Reply:
I see, then it's good.